deploy/podsecuritypolicy.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: operator
+spec:
+  allowPrivilegeEscalation: false
+  allowedCapabilities:
+  - '*'
+  volumes:
+  - '*'
+  hostNetwork: true
```

Comment:
Does this need hostNetwork, hostPorts, hostIPC, hostPID?
Also can we restrict the volumes and allowedCapabilities?

Reply:
No, this parameters aren't required. Updated.